cmd/preupgradechecks/main.go

```diff
@@ -71,5 +72,10 @@ Options:
 		return
 	}
 
+	err = crdBackedClient.LatestSchemaApplied()
+	if err != nil {
+		logger.Info("New CRDs are not applied")
```

Comment:
Please use `logger.Fatal`.  It does error logging as well as exit.
Also  errors should small letter than capital letter.

```
logger.Fatal("new CRDs are not applied yet")
```